tensorflow/python/ops/image_ops_impl.py

```diff
@@ -792,6 +792,15 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     ValueError: If the shape of `image` is incompatible with the `offset_*` or
       `target_*` arguments, or either `offset_height` or `offset_width` is
       negative.
+      
+  Usage example:
+  ```python
+    
+    >>> x = tf.random.uniform((28, 28, 1))
+    >>> tf.image.pad_to_bounding_box(x, 28, 28, 100, 100).shape
+    >>> # TensorShape([100, 100, 1])
```

Comment:
Why is the result commented? You can remove the `>>> #` and this will make the doctest work.

Reply:
Did it!